livechat/customer/rtm/client.py

```diff
@@ -623,5 +624,10 @@ def update_event_properties(self,
 
 
 class CustomerRTM33(CustomerRTMInterface):
-    ''' CustomerRTM version 3.3 class. '''
+    ''' Customer RTM version 3.3 class. '''
+    pass
+
+
+class CustomerRTM34(CustomerRTMInterface):
+    ''' Customer RTM version 3.4 class. '''
     pass
```

Comment:
pass is not necessary (https://www.python.org/dev/peps/pep-0257/)

Reply:
👍